src/app/child-dev-project/notes/model/note.ts

```diff
@@ -107,6 +109,7 @@ export class Note extends Entity {
     label: $localize`:Label for the social worker(s) who created the note:SW`,
     dataType: "entity-array",
     additional: User.ENTITY_TYPE,
+    defaultValue: PLACEHOLDERS.CURRENT_USER,
```

Comment:
would this need special handling to generically work for any innerDataType within an array?